### DISCUSSION TITLE
Sorting on attachments

### DISCUSSION THREAD

**Discussion by baknight1975** [cerb/cerb-release#30] [Q&A] [answered]
One of our users has inquired if we can sort worklists based on attachments.   Is this something that is supported.   We've take a look at the sort option and don't see a way to use links.attachments in this manner.  Can someone confirm if this type of sort is possible?

Thanks,'
Barry
Manager, IT Customer Service
Fleming College

**Comment by Hildy** [comment, marked as answer]
Barry, do you mean you want to have a list that only shows tickets with attachments?  You can use a deep search on messages for that: `messages:(attachments:(id:>0))` (other attachment properties are also supported there).

If that's not what you mean, can you clarify, please?

**Reply by baknight1975** [reply]
Thanks Dan.   They are looking to sort a work list with all the tickets with attachments at the top of the list.   Wasn't sure if this was possible.   I will however see if this would work for them.  They can just create another work list with attachments only tickets.  Thanks for the quick reply
